src/components/custom-field-input-date/custom-field-input-date.md

```diff
@@ -0,0 +1,61 @@
+A simple date field for custom fields. Permits a variety of formats and allows custom validation.
+
+##### Basic usage:
+
+```jsx
+<CustomFieldInputDate
+    id="my-birthday-1"
```

Comment:
Looks like we're diverging from the other markdown files: indentation is 1-tab which is 2-spaces. This codebase is small enough that I want to keep up some resemblance of spacing consistency.